spec/namespaces/list.yaml

```diff
@@ -0,0 +1,485 @@
+      style: form
+    list.indices::query.help:
+      name: help
+      in: query
+      description: Return help information.
+      schema:
+        type: boolean
+        default: false
+        description: Return help information.
+    list.indices::query.include_unloaded_segments:
+      in: query
+      name: include_unloaded_segments
+      description: If true, the response includes information from segments that are not loaded into memory.
+      schema:
+        type: boolean
+        default: false
+      style: form
+    list.indices::query.local:
+      name: local
+      in: query
+      description: Return local information, do not retrieve the state from cluster-manager node.
+      schema:
+        type: boolean
+        default: false
+        description: Return local information, do not retrieve the state from cluster-manager node.
+    list.indices::query.master_timeout:
+      name: master_timeout
+      in: query
+      description: Operation timeout for connection to master node.
+      schema:
+        $ref: '../schemas/_common.yaml#/components/schemas/Duration'
+      x-version-deprecated: '2.0'
+      x-deprecation-message: To promote inclusive language, use 'cluster_manager_timeout' instead.
+      deprecated: true
+    list.indices::query.pri:
+      in: query
+      name: pri
+      description: If true, the response only includes information from primary shards.
+      schema:
+        type: boolean
+        default: false
+      style: form
+    list.indices::query.s:
+      name: s
+      in: query
+      description: Comma-separated list of column names or column aliases to sort by.
+      style: form
+      schema:
+        type: array
+        items:
+          type: string
+        description: Comma-separated list of column names or column aliases to sort by.
+      explode: true
+    list.indices::query.time:
+      in: query
+      name: time
+      description: The unit used to display time values.
+      schema:
+        $ref: '../schemas/_common.yaml#/components/schemas/TimeUnit'
+      style: form
+    list.indices::query.v:
+      name: v
+      in: query
+      description: Verbose mode. Display column headers.
+      schema:
+        type: boolean
+        default: false
+        description: Verbose mode. Display column headers.
+    list.indices::query.next_token:
+      name: next_token
+      in: query
+      description: Token to retrieve next page of indices.
+      schema:
+        type: ['null', string]
+        description: Token to retrieve next page of indices.
+    list.indices::query.size:
+      name: size
+      in: query
+      description: Maximum number of indices to be displayed in a page.
+      schema:
+        type: integer
+        format: int32 
+    list.indices::query.sort:
+      name: sort
+      in: query
+      description: Defines order in which indices will be displayed. If "desc", most recently created indices would be displayed first.
```

Comment:
I think we quote these as `desc`, but double check.

Reply:
Thanks for pointing out. Replaced "desc" with `desc` as is being used at other places.